packages/orbit-design-tokens/src/dictionary/utils/is.ts

```diff
@@ -6,7 +6,7 @@ const isTypeOf = ({ type, category }: DesignToken, typeName: string): boolean =>
 export const isColor = (prop: DesignToken): boolean => isTypeOf(prop, "color");
 
 export const isColorPalette = (prop: DesignToken): boolean =>
-  isColor(prop) && prop.attributes.name.startsWith("palette");
+  isColor(prop) && prop.path.includes("palette");
```

Comment:
Why is this change necessary? It seems like all the color palette tokens do start with `palette`.

Reply:
I feel like it's better to get them via path, rather than name and attributes.

Reply:
Okay, but is it contributing to the success of this contribution? If not, I would say it's out of scope and could/would be introduced in a different PR as it is a distraction.